.prettierignore

```diff
@@ -0,0 +1 @@
+tailwind.config.js
```

Comment:
not sure why this was added

Reply:
Prettier was messing up the syntax of tailwind config for some reason